docs/catalog.md

```diff
@@ -0,0 +1,117 @@
+# What is the LEAP Data Catalog?
+
+The LEAP Data Catalog is a centralized repository that consolidates various LEAP datasets into a single JSON file. This JSON catalog is generated from individual YAML files, each representing a dataset. The catalog is currently in development, and a draft version is available at [LEAP Data Catalog](https://leap-data-catalog.vercel.app/).
+
+## The Schema
+
+The catalog is generated from individual YAML files. Each dataset, or feedstock, needs two files: `meta.yaml` and `catalog.yaml`. These files can be located in different locations to separate metadata curated during dataset creation (like Pangeo-Forge recipes) from catalog information that enhances the metadata for the LEAP catalog.
```

Comment:
Maybe a RW? -> ...or feedstock, needs two files. A `meta.yaml` file and  a`catalog.yaml` file.

Reply:
I think you can cut located. RW -> These files can be in different locations...

Reply:
Not super clear on what this means. Mayce we can cut? `that enhances the metadata for the LEAP catalog.`